projects/schematics/src/add-spartacus/index.ts

```diff
@@ -411,6 +413,9 @@ function prepareSpartacusFeatures(options: SpartacusOptions): string[] {
     ...(shouldAddFeature(CLI_TRACKING_FEATURE, options.features)
       ? [SPARTACUS_TRACKING]
       : []),
+    ...(shouldAddFeature(CLI_USER_FEATURE, options.features)
```

Comment:
Actually for user lib we want something different. We don't want to even ask user about it. We should just directly install it (both account and profile).